### PR TITLE
Mobile E2E tests: room invitations

### PR DESCRIPTION
**Pull request by alexandra-marin**
#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/748/mobile-tests-room-invitations
#### Testing instructions  
```
npm run test-invites-android
npm run test-invites-ios
```
Get new credentials from https://github.com/PeerioTechnologies/internal-documentation/blob/master/frontend/e2e-account-credentials.md and paste them in you local env.sh

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

**Comment by seavan** [comment]
@alexandra-marin needs to be merged with dev (there are conflicts now)

my environment runs android tests fine, see `npm run test-profile-settings-android` on dev.

**Comment by flohdot** [comment]
lgtm :)

File: test/code/steps/rooms.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    When('I cancel the invite', async function () {
+        await this.chatListPage.scrollDownHelper(); // room can be at the bottom of the list
+        await this.chatListPage.roomWithTitle(this.roomName).click();
+
+        await this.app.pause(5000); // android needs a pause
+        await this.chatPage.roomWithTitle(this.roomName).click();
+        await this.chatPage.invitedContactMore.click();
+        await this.chatPage.removeInvitedButton.click();
+        
+        await this.app.pause(5000); // timeout for invited contact to disappear from list
```
Comment (seavan) [inline test/code/steps/rooms.js]: no long timeouts please, use "find"

File: test/code/steps/rooms.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        await this.chatListPage.roomWithTitle(this.roomName).click();
+
+        await this.roomInvitePage.declineButton.click();
+    });
+
+    When('I cancel the invite', async function () {
+        await this.chatListPage.scrollDownHelper(); // room can be at the bottom of the list
+        await this.chatListPage.roomWithTitle(this.roomName).click();
+
+        await this.app.pause(5000); // android needs a pause
```
Comment (seavan) [inline test/code/steps/rooms.js]: milana-source-files.zip

**Comment by flohdot** [comment]
Ah, just thought of another scenario — or two I guess: can be re-invited to a room after declining or leaving.

**Comment by alexandra-marin** [comment]
@seavan can you post more logs? what is the reason for failing?

**Comment by seavan** [comment]
it fails immediately after taking screenshot. screenshot is a success (flag is set correctly in java).

after it fails, the process is restarted.

```
[debug] [MJSONWP] Calling AppiumDriver.getScreenshot() with args: ["3db9fa3b-c6bc-4248-a864-d8866a97b112"]
[debug] [ADB] Device API level: 23
[debug] [MJSONWP] Responding to client with driver.getScreenshot() result: "iVBORw0KGgoAAAANSUhEUgAABaAAAAoACAYAAABH9IXPAACAAElEQVR4AezBf4zfhUH/8efrc5+jV6DcIQwky41GxgT8LnzwBwiiPW2mDOXDJ1s3dAb4FCebiY6DocD+gHYIJGTS4nBxMcAdbCxsTPqZDHVxoSURky2mdwSDuhCvNhuB8OM+Lf1Be3evL+/3h3G0Xtmgd/Tu+no8BJiIiIiIiIiIiIiIiFlWISIiIiIiIiIiIiJiDlSIiIiIiIiIiIiIiJgDFSIiIiIiIiIiIiIi5kCFiIiIiIiIiIiIiIg5UCEiIiIiIiIiIiIiYg5UiIiIiIiIiIiIiIiYAxUiIiIiIiIiIiIiIuZAhYiIiIiIiIiIiIiIOVAhIiIiIiIiIiIiImIOVIiIiIiIiIiIiIiImAMVIiIiIiIiIiIiIiLmQIWIiIiIiIiIiIiIiDlQISIiIiIiIiIiIiJiDlSIiIiIiIiIiIiIiJgDFSIiIiIiIiIiIiIi5kCFiIiIiIiIiIiIiIg5UCEiIiIiIiIiIiIiYg5UiIiIiIiIiIiIiIiYA1UiIiIiIiIiIiLmme7ubvbu3ctCccopp9DX18fBePbZZ3n++eeZmJggYrGoEhERERERERGxyC1fvpyTTz6ZgYEB+vr6qNVqFGq1Gn19fRzI+Pg4IyMjFEZGRhgfH2fjxo1s2bKFsbExYm7UajW2bt3Kiy++yEJx7LHHctJJJ3Ewtm/fzgsvvEDEYlIlIiIiIiIiImKRWbFiBQMDAwwMDFCr1ejr6+Od6OvrY2BggMLAwABvNj4+zsjICBs3bmTjxo1s2rSJOHi1Wo3+/n62bt1KRCx8VSIiIiIiIiIiFri+vj4uvvhiGo0GAwMD9PX1Mdf6+voYGBhgYGCAwvj4OBs3bmTDhg20Wi3Gx8eJt6dWq9Hf309ELB5VIiIiIiIiIiIWqMsvv5xGo0Gj0eBQ6+vro9Fo0Gg0KGzYsIENGzYwPDxM/GS1Wo3+/n4iYnGpEhERERERERGxgCxf...
[HTTP] <-- GET /wd/hub/session/3db9fa3b-c6bc-4248-a864-d8866a97b112/screenshot 200 3116 ms - 2642950
[HTTP] --> POST /wd/hub/session/3db9fa3b-c6bc-4248-a864-d8866a97b112/appium/device/remove_app {"bundleId":"com.peerio.app"}
[debug] [MJSONWP] Calling AppiumDriver.removeApp() with args: ["com.peerio.app",null,null,null,"3db9fa3b-c6bc-4248-a864-d8866a97b112"]
```

**Pull request merged**